language/stdlib/modules/LibraWriteSetManager.move

```diff
@@ -75,6 +77,10 @@ module LibraWriteSetManager {
             PROLOGUE_EINVALID_ACCOUNT_AUTH_KEY
         );
     }
+    spec fun prologue {
+        pragma aobrts_if_is_partial = true;
```

Comment:
Spell error, no reported right now! Means also you don't need it (because not aborts_if specs)

Reply:
Done.